pom.xml

```diff
@@ -209,7 +209,6 @@
         <executions>
           <execution>
             <id>attach-javadocs</id>
-            <phase>deploy</phase>
```

Comment:
You removed this because it's the default, correct?

Reply:
No, both javadoc and sources plugins bind by default to `package` phase and that's how we want it.

Reply:
Does this configuration make the build always generate Javadocs at 'package' phase?

Reply:
> Does this configuration make the build always generate Javadocs at 'package' phase for every module?

True! I changed that so now we need to explicitly specify for which project we want to generate sources and javadoc.
We could also move those two plugins to `deploy` profile (where we have maven-gpg-plugin) if we don't need them unless we deploy.
But I think we could leave it be as we will probably change our deployment process (using maven release plugin).